Spigot-Server-Patches/0003-MC-Dev-fixes.patch

```diff
@@ -71,8 +71,39 @@ index 650ef14758..35aea4829f 100644
          this.emittedLight = new NibbleArray();
          if (flag) {
              this.skyLight = new NibbleArray();
+diff --git a/src/main/java/net/minecraft/server/DataPaletteBlock.java b/src/main/java/net/minecraft/server/DataPaletteBlock.java
+index 20b1bc1b4..590794d85 100644
+--- a/src/main/java/net/minecraft/server/DataPaletteBlock.java
++++ b/src/main/java/net/minecraft/server/DataPaletteBlock.java
+@@ -82,7 +82,7 @@ public class DataPaletteBlock<T> implements DataPaletteExpandable<T> {
+         int j;
+ 
+         for (j = 0; j < databits.b(); ++j) {
+-            Object object = datapalette.a(databits.a(j));
++            T object = (T) datapalette.a(databits.a(j)); // Paper - decompile fix
```

Comment:
Sorry. Can you do one more please? Change `DataPalette datapalette` to `DataPalette<T> datapalette` and remove the cast.

Reply:
ok